config/printer-creality-ender3-s1-2021.cfg

```diff
@@ -109,18 +109,23 @@ probe_with_touch_mode: true
 stow_on_each_sample: false
 
 [bed_mesh]
-speed: 120
+speed: 100
 mesh_min: 20, 20
-mesh_max: 200, 200
-probe_count: 4,4
+mesh_max: 200, 183
+probe_count: 5,5
 algorithm: bicubic
 
+[bed_screws]
+screw1: 20, 25
+screw2: 195, 25
+screw3: 195, 195
+screw4: 20, 195
+
 [safe_z_home]
 home_xy_position: 147, 154
 speed: 75
-z_hop: 5
+z_hop: 10
 z_hop_speed: 5
-move_to_previous: true
```

Comment:
Removed to keep the probe to the centre of the bed and, in this way, being able to execute correctly the `PROBE_CALIBRATE` command